projects/sandbox/vizapp/vizapp/vetoes.py

```diff
@@ -0,0 +1,74 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List
+
+import numpy as np
+from gwpy.segments import DataQualityDict
+
+# TODO: gating should really be applied directly to the strain
+
+
+CATEGORIES = ["CAT1", "CAT2", "CAT3", "GATES"]
+
+
+def gates_to_veto_segments(path: Path):
+    """Naively convert gate files to vetoes segments"""
+    gates = np.loadtxt(path)
+    centers = gates[:, 0]
+    windows = gates[:, 1]
+    tapers = gates[:, 2] + 0.375
+
+    vetoes = np.array(
+        [
+            [center - window - taper, center + window + taper]
+            for center, window, taper in zip(centers, windows, tapers)
+        ]
+    )
+
+    return vetoes
+
+
+@dataclass
+class VetoeParser:
```

Comment:
Pedantic thing for here and elsewhere, but shouldn't the singular just be "veto"?